docker/Dockerfile

```diff
@@ -138,4 +138,4 @@ EXPOSE ${APP_PORT}
 COPY ./docker/entry.sh $MEALIE_HOME/run.sh
 
 RUN chmod +x $MEALIE_HOME/run.sh
-ENTRYPOINT $MEALIE_HOME/run.sh
+ENTRYPOINT ["/app/run.sh"]
```

Comment:
This had to be switched to the JSON type format because of the way docker handles the starting of the container scripts which also doesn't allow us to use env variables in the string, so that's neat!